dmk2cw.man

```diff
@@ -229,6 +229,28 @@ disk intact.  With fill = 0x1nn, the fill pattern is 0xnn encoded
 in FM.  With fill = 0x2nn, the fill pattern is 0xnn encoded in
 MFM.
 .TP
+.B \-a \fIrate_adj\fP
+Causes dmk2cw to multiply the average data rate by rate_adj (default
+1.0).  Values should be close to 1.0; for example, 1.005 makes the
+data rate 0.5 percent faster, while 0.995 makes it 0.5% slower.
+Making the data rate 1% faster has a similar effect to running the
+drive motor %1 slower: more data fits on the track.  Of course if the
```

Comment:
In this paragraph, you refer to "0.5 percent faster", "0.5% slower" and "%1".  Would prefer a little more consistency.  When you get a chance in a later commit, please fix "0.5 percent" to just "0.5%" and "%1" to 1%".